lib/rucio/core/rule.py

```diff
@@ -1303,6 +1304,9 @@ def update_rule(rule_id, options, session=None):
             if key == 'source_replica_expression':
                 rule.source_replica_expression = options['source_replica_expression']
 
+            if key == 'comment':
+                rule.comments = options['comment']
```

Comment:
is this called 'comments' in the rule definition? hmmmm. that's a weird inconsistency. @bari12 what do you think?

Reply:
yes, 
https://github.com/rucio/rucio/blob/4c5f830d048a0bea9c1c3e078eceb244d5946b7a/lib/rucio/db/sqla/models.py#L954

And changing it to "comment**s**" will demand changes in the comment option of `add-rule` command as well. But will surely be less involved than changing the rule definition to "comment".

Reply:
One option is to change the client API to add `comments` as an alias of `comment` and eventually send a deprecation warning for the `comment` field name.

Reply:
I think that is fine, to change it in the database is a bit more complicated obviously. But the alias is good.